src/help.ts

```diff
@@ -76,9 +76,42 @@ export default class BalenaHelp extends Help {
 	public async showCommandHelp(command: Command.Loadable): Promise<void> {
 		const name = command.id;
 		const depth = name.split(':').length;
+		// We are suppressing how oclif prints sub-commands & sub-topics
+		// so that we can add a header for them right after.
```

Comment:
It looks like oclif should be adding a header already: https://github.com/oclif/core/blob/ec07566d96fb96836cb1d6a0533bab0fba6febfc/src/help/index.ts#L167-L184

I know we did not see it, but we should see if we can figure out why we don't see it. And if we do need to override how the subtopics are displayed in order to attach our own header, I expect we should be able to just override `formatTopics` instead of dealing with the suppression and mirroring